astro/src/content/quickstarts/quickstart-ruby-rails-web.mdx

```diff
@@ -10,8 +10,10 @@ icon: /img/icons/ruby-on-rails.svg
 faIcon: fa-gem
 color: rose
 ---
-import Aside from '../../components/Aside.astro';
-import RemoteCode from '../../components/RemoteCode.astro';
+import Aside from '/src/components/Aside.astro';
```

Comment:
```
import Aside from '../../components/Aside.astro';
import LoginAfter from '../../diagrams/quickstarts/login-after.astro';
import LoginBefore from '../..//diagrams/quickstarts/login-before.astro';
import RemoteCode from '../../components/RemoteCode.astro';
```
relative imports will resolve properly in the editor

Reply:
good point.  at least for in the astro components.  My Intellij doesn't know what to do with them in .mdx files